Formula/lilypond.rb

```diff
@@ -0,0 +1,63 @@
+class Lilypond < Formula
+  desc "Music engraving program"
+  homepage "https://lilypond.org"
+  url "https://lilypond.org/download/sources/v2.22/lilypond-2.22.1.tar.gz"
+  sha256 "72ac2d54c310c3141c0b782d4e0bef9002d5519cf46632759b1f03ef6969cc30"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url "https://lilypond.org/source.html"
+    regex(/href=.*?lilypond[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  depends_on "bison" => :build # Lilypond requires bison 2.4.1 or above
+  depends_on "fontforge" => :build
+  depends_on "gettext" => :build
+  depends_on "glib" => :build
+  depends_on "perl" => :build
+  depends_on "pkg-config" => :build
+  depends_on "t1utils" => :build
+  depends_on "texinfo" => :build # lilypond requires texinfo 6.1 or above
+  depends_on "texlive" => :build
+
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "ghostscript"
+  depends_on "guile@2"
+  depends_on "pango"
+  depends_on "python@3.9"
+
+  uses_from_macos "flex" => :build
+
+  def install
+    system "./configure",
+            "--prefix=#{prefix}",
+            "--datadir=#{share}",
+            "--with-texgyre-dir=#{Formula["texlive"].opt_share}/texmf-dist/fonts/opentype/public/tex-gyre",
+            "--disable-documentation"
+    ENV.prepend_path "LTDL_LIBRARY_PATH", Formula["guile@2"].lib
+    system "make"
+    system "make", "install"
+
+    elisp.install share.glob("emacs/site-lisp/*.el")
+
+    libexec.install bin/"lilypond"
+
+    (bin/"lilypond").write_env_script opt_libexec/"lilypond",
```

Comment:
Should this be

```ruby
(bin/"lilypond").write_env_script libexec/"lilypond",
```

(not `opt_libexec/"lilypond"`), or am I missing something?

Reply:
They're almost always the same, but `libexec` is a better choice for some edge cases. Let's do that.

Reply:
In previous version, we used `write` statement instead of `write_env_script`, but the script is the same.